nengo/exceptions.py

```diff
@@ -60,6 +60,17 @@ def __str__(self):
             if self.url is not None else "")
 
 
+class MovedError(NengoException):
```

Comment:
I wonder if this should inherit from `ObsoleteError` because both exceptions mark a “feature that has been removed in a backwards-incompatible way”, but in this case it can also be found at another place.

Reply:
Looking more at it, `MoveError` has a significantly different message than `ObsoleteError`, so better to keep them separate IMO.